src/Resources/views/form/forms.html.twig

```diff
@@ -77,20 +92,30 @@
                                     <td data-search="">
                                         <div class="btn-group">
                                             {% for action in table.itemActions %}
+                                                {%  if action.dynamic == true %}
+                                                    {% set routeParameters = {} %}
+                                                    {% for name, property in action.routeParameters %}
+                                                        {% set routeParameters = routeParameters|merge({ (name): attribute(line.data, property) }) %}
+                                                    {%  endfor %}
+                                                {% else %}
+                                                    {% set routeParameters = action.routeParameters|merge({(table.attributeName): line.data.id}) %}
+                                                {% endif %}
                                                 {% if action.post %}
-                                                    {% include '@EMSCore/elements/post-button.html.twig' with {
-                                                        'url': path(action.route, action.routeParameters|merge({(table.attributeName): line.data.id})),
-                                                        'message': action.messageKey,
-                                                        'label': action.labelKey,
-                                                        'itemLabel': attribute(line.data, table.labelAttribute),
-                                                        'icon': action.icon
-                                                    }%}
+                                                    {% if action.route != 'EMS_user_apikey' or 'ROLE_API' in line.data.roles  %}
```

Comment:
Can you explain this line

Reply:
@theus77 I don't want the post-button to show if it concerns the "Generate API Key" button, unless if the user has the role ROLE_API.